Source/Core/Core/PowerPC/JitCommon/JitBackpatch.cpp

```diff
@@ -71,6 +71,22 @@ bool Jitx86Base::BackPatch(u32 emAddress, SContext* ctx)
 		return false;
 	}
 
+	auto it2 = pcAtLoc.find(codePtr);
+	u32 pc;
+	if (it2 == pcAtLoc.end())
+	{
+		if (info.isMemoryWrite || jit->js.memcheck)
+		{
+			PanicAlert("BackPatch: no pc entry for address %p", codePtr);
```

Comment:
Are you sure? I don't see where the code in Jit.cpp writes the PC to SPR_DAR, but I might be mistaken.